geomesa-tools/conf/sfts/marinecadastre-ais/README.md

```diff
@@ -0,0 +1,49 @@
+# Marine Cadastre AIS
```

Comment:
Would prefer a link to the documentation but don't let this block.

Reply:
Yeah, I was lazy and just copied the existing converter docs